src/features/home/ui/TopFiveStacks.tsx

```diff
@@ -0,0 +1,36 @@
+import { useEffect, useState } from 'react';
+import * as S from './topFiveStacks.style';
+import { fetchData } from '@/features/home/api/topFiveStacks.api';
+import { StackData } from '@/features/home/model/topFiveStacks.model';
+
+const TopFiveStacks = () => {
+  // TODO : top5 axios get 요청 코드
+  const [topFiveData, setTopFiveData] = useState<StackData[] | undefined>();
+
+  useEffect((): void => {
+    fetchData()
+      .then((data) => {
+        setTopFiveData(data);
+      })
```

Comment:
이렇게 표현해줘도 괜찮을 것 같네요~
```suggestion
      .then(setTopFiveData)
```

Reply:
https://github.com/GrebbMe/grabbme-fe/pull/87/commits/6891fdb4d37555b00cba96ea9de121d15c13f721 에서 반영했습니다!!